include/openrave/sensor.h

```diff
@@ -324,9 +329,20 @@ class OPENRAVE_API SensorBase : public InterfaceBase
             if (!pOther) {
                 return false;
             }
-            return SensorGeometry::operator==(other);
+            return SensorGeometry::operator==(other)
+                   && polarity == pOther->polarity
+                   && correction_matrix == pOther->correction_matrix;
         }
+
+        bool SerializeXML(BaseXMLWriterPtr writer, int options=0) const override;
+        bool SerializeJSON(rapidjson::Value& value, rapidjson::Document::AllocatorType& allocator, dReal fUnitScale=1.0, int options=0) const override;
+        bool DeserializeJSON(const rapidjson::Value& value, dReal fUnitScale=1.0) override;
+
+        int polarity; ///< sensor's polarity. set +1 if load is attached to the side of sensor which sensor's spec explains. set -1 if load is attached to the opposite side.
+        boost::array<dReal,36> correction_matrix; ///< correction matrix. force -> torque order.
```

Comment:
@eisoku9618 please have better description of how `correction_matrix` should be used.

Reply:
thank you for your review. I addressed it by https://github.com/rdiankov/openrave/pull/912/commits/bb5cc970aca84bf445a972f7fb547f902bf2c7e1